### ISSUE TITLE
Zebra Stripes

### ISSUE THREAD

**Issue by Broxzier** [OpenRCT2/OpenRCT2#7552]
In PR #7548 the question about using striped lists was raised again. We're already using stripes lists for some windows, but they have also been removed. I'd like to get a clear decision about whether or not we should be using striped lists.

After searching for different styles I found this research: http://alistapart.com/article/zebrastripingmoredataforthecase

![](https://i.stack.imgur.com/WTCMR.gif)
![](https://i.stack.imgur.com/509AZ.gif)

This research shows that with having both the highest "preferred" and lowest "least preferred" percentage, it's quite clear that the majority is in favour of striped tables. Whilest this research was about tables, I think it's still very related to our lists with mutliple columns.

On the other hand, there's this article that basically says all formatting should be removed: https://www.darkhorseanalytics.com/blog/data-looks-better-naked (there's a video slideshow on this page)

And then there is [Gestalt's law of enclosure](https://blog.fusioncharts.com/how-to-use-the-gestalt-principles-for-visual-storytelling-podv/), which gives the user more of a sense that cells belong together. This is exactly what a striped background achieves.

![](https://www.fusioncharts.com/blog/wp-content/uploads/2014/03/illustrations.jpg)

Personally I'm in favour of using a striped background for lists with multiple columns. What do you guys think?

**Comment by deurklink** [comment]
Could you maybe show an example of what this would look like in the tile explorer? (i dont know how much work it would take)

**Comment by Broxzier** [comment]
Not the tile inspector window, but here are two screenshots from https://github.com/OpenRCT2/OpenRCT2/pull/7420:

|With zebra stripes|Without zebra stripes|
|----|----|
|![](https://user-images.githubusercontent.com/604665/38830879-ef1a2178-41bd-11e8-8459-918bf1f8e717.png)|![](https://user-images.githubusercontent.com/604665/38827937-c3cff7ee-41b4-11e8-8722-9980fd766860.png)|

**Comment by pizza2004** [comment]
Why is SLCT darker on the zebra stripes?

**Comment by Broxzier** [comment]
Probably because it was the last item that was hovered over.

**Comment by marijnvdwerf** [comment]
This research talks about non-interactive tables; such as the one seen in the finances window. While I can understand your desire for striped tables, I think that requires a significantly different highlight.

Also, striping the row that is being hovered makes a lot less sense if zebra-striping is applied.

![image](https://user-images.githubusercontent.com/737603/40873862-50ae0bf6-6668-11e8-8e7d-c9742f1a56d5.png)


Nevertheless, I'd rather see an experiment with row dividers instead, as that would at least not adversely affect the contrast between foreground and background, as we already do a horrible job at keeping decent contrast.

**Comment by Broxzier** [comment]
@marijnvdwerf Do you mean like this? (horizontal stripes and selected item has a darker background)

![the openrct2 group park 8 2018-06-02 14-25-41](https://user-images.githubusercontent.com/9705046/40874551-238057d4-6671-11e8-83c5-7c3f852be575.png)

**Comment by marijnvdwerf** [comment]
Yes, like that. Though the list items could definitely do with some extra padding; also increasing their target size.

**Comment by Broxzier** [comment]
It's the same as the Lined version from the image above, and while it has the same amount of least preferred votes, it has a lower score for preferred, so I still think striped lists is the way to go. I do like the higher contrast for the selected row though.

**Comment by Krutonium** [comment]
I prefer it with the Zebra stripes - it helps convey that they information goes together, and also allows easier following of said data.

**Comment by SijmenSchoon** [comment]
Another way of improving readability is to increase the row spacing, which looks much calmer than zebra stripes. The obvious disadvantage is that less data fits on the screen.
